node/tracing-logging-and-metrics-for-sdk-and-core.md

```diff
@@ -0,0 +1,96 @@
+## Tracing, Logging, and Metrics in the NodeJS SDK and Core
+
+### What's already implemented in the Alpha release?
+
+Core [uses opentelemetry](https://github.com/temporalio/sdk-core#debugging) for generating logs and traces, it does not emit any metrics.
+
+The Node SDK uses `WorkerOptions.logger` directly for logging and the opentelemetry API for tracing with no output by default, it does not emit any metrics either.
+
+There's an example of how to export the traces to jaeger in [`bench.ts`](https://github.com/temporalio/sdk-node/blob/a25c0feca791a195eb5e50cf8c0fd9e0b5a1db83/packages/test/src/bench.ts#L105).
+
+See the [documentation](https://docs.temporal.io/docs/node/logging/) on how to customize the Worker logger.
+
+There's currently not a way to log from Workflow internals (inside the isolate).
+
+### Requirements
+
+- Link traces between Node and Core for a more complete picture
+- Core traces should be propagated through NodeJS
+- Core metrics should be propagated through NodeJS
+- Core logs should integrate with the Worker logger
+- Support logging from Workflow internals
+- Support logging in Worker either within an otel span or without context (e.g. Worker state changed)
+- Metrics should only be used when no trace context is available
```

Comment:
Why is that?

Reply:
I mean don't do things that you can already get with tracing with metrics e.g: https://github.com/temporalio/samples-go/blob/master/metrics/metrics.go

Reply:
Will update the doc